tests/actions.go

```diff
@@ -1458,13 +1448,19 @@ func (oa *operatorActions) getBackupDir(info *TidbClusterInfo) ([]string, error)
 	}
 
 	cmd := fmt.Sprintf("kubectl exec %s -n %s ls /data", getBackupDirPodName, info.Namespace)
-	glog.Infof(cmd)
+
+	time.Sleep(20 * time.Second)
```

Comment:
Can we poll the following condition so we can proceed on earlier if condition meet?

Reply:
I try to poll at first but the time is hard to controller in one  minute. for every minute generate a cronjob to backup. At last we have to check the backup number.